src/main/resources/application.yaml

```diff
@@ -75,8 +75,8 @@ aurora:
 
 logging:
     level:
-        no.skatteetaten: INFO
-        no.skatteetaten.aurora.boober: INFO
+        no.skatteetaten: DEBUG
+        no.skatteetaten.aurora.boober: DEBUG
```

Comment:
Dette er allerede angitt for utv04 i config: 
```
  LOGGING_LEVEL_NO_SKATTEETATEN: DEBUG
  LOGGING_LEVEL_NO_SKATTEETATEN_AURORA_BOOBER: DEBUG
```

Reply:
Ja ok, men vi skal teste i utv

Reply:
Da kan du legge på ovenstående config der 😉